guides/plugins/plugins/index.md

```diff
@@ -9,6 +9,23 @@ nav:
 
 Shopware plugins are extensions that enhance the functionality and features of the Shopware e-commerce platform. Plugins are designed to extend the core capabilities of Shopware and offer additional functionalities that are not available out of the box. While apps and themes are also extensions but they differ from plugins. To better understand the differences, take a look at the [Overview](../../../guides/plugins/overview) article.
 
+## Feature Comparison
+
+::: tip
+
+For projects customizations, it is recommended to use Bundles instead of plugins. As bundles are not managed via Administration and don't have lifecycle they offer full control over the project.
```

Comment:
```suggestion
For customizing projects, it is recommended to use bundles instead of plugins.
As bundles are not managed via Administration and don't have a lifecycle they offer full control over the project.
```